lib/WP_Auth0_LoginManager.php

```diff
@@ -146,15 +135,16 @@ public function init_auth0() {
 
 		// Catch any incoming errors and stop the login process.
 		// See https://auth0.com/docs/libraries/error-messages for more info.
-		if ( $this->query_vars( 'error' ) || $this->query_vars( 'error_description' ) ) {
-			$error_msg  = sanitize_text_field( rawurldecode( $_REQUEST[ 'error_description' ] ) );
-			$error_code = sanitize_text_field( rawurldecode( $_REQUEST[ 'error' ] ) );
+		if ( ! empty( $_REQUEST['error'] ) || ! empty( $_REQUEST['error_description'] ) ) {
```

Comment:
would an `error_description` by itself be valid? I guess only when `error` is present. So 2nd clause could be removed.